src/ADDSMock/service-configuration/scs/basic-catalogue.cs

```diff
@@ -0,0 +1,129 @@
+using ADDSMock.Domain.Mappings;
+using System.Net;
+using WireMock.Matchers;
+using WireMock.RequestBuilders;
+using WireMock.ResponseBuilders;
+
+public void RegisterFragment(WireMockServer server, MockService mockService)
+{
+    var urlPattern = ".*/v2/catalogues/s100/basic.*";
+
+    server
+        .Given(
+            Request.Create()
+                .WithUrl(new RegexMatcher(urlPattern))
+                .UsingGet()
+        )
+        .RespondWith(
+            Response.Create()
+                .WithStatusCode(200)
+                .WithHeader("Content-Type", "application/json")
+                .WithHeader("Last-Modified", DateTime.UtcNow.ToString("yyyy-MM-ddTHH:mm:ssZ"))
+                .WithHeader("_X-Correlation-ID", "200-ok-guid-scs-basic-catalogue")
```

Comment:
Please declare "Content-Type", "Last-Modified", "_X-Correlation-ID" & "application/json" as constant and use it.